osquery/osquery.go

```diff
@@ -51,14 +54,24 @@ type OsqueryConfig struct {
 type OsqueryResultLog struct {
 	Name           string `json:"name"`
 	HostIdentifier string `json:"hostIdentifier"`
-	UnixTime       string `json:"unixTime"`
+	UnixTime       int    `json:"unixTime"`
```

Comment:
This is an osquery 3.0 change I believe. Epoch and Counter would also have been a 3.0 change, but this struct didn't have them before. IMO we should just make these ints.